spec/features/post_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe "Post" do
 
-  let(:user) { FactoryGirl.create :user }
+  let!(:user) { build :user }
```

Comment:
I realize now that because build does not persist to the database, validations would not run. It does not really do me much good to go with build in this case, so I'll go ahead and change that.